net/i2pchat/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+qt5.depends_component qtmultimedia qtsvg
+
+# TODO: find a version working with Qt4.
+# https://github.com/vituperative/i2pchat/issues/31
+github.setup        vituperative i2pchat 91260180ca22dafdaa0e82fb33e148c0b26cb092
+version             2022.05.22
```

Comment:
the usual question: why not a tagged version? Most changes since the tagged version 0.2.37 seem irrelevant as they only touch upstreams CI systems, some small updates to .png files and very-very few actual changes in the code. Does this really not work with the released version?

Reply:
Well, whenever there is a gap in years, I prefer newer code (unless it is explicitly marked as ABI-breaking or experimental) to stylistic convention. Especially when nothing else depends on a port.

But since this is for Qt5-having systems only, not a case where I would spend effort trying to convince anyone. If you insist, I can switch to an archaic tag, be it.

Reply:
same as in other ports, if changes were substantial and no tags in years, then it's perhaps time to ask upstream to release a new version. Alternatively, we could just stop adding ports where upstream is not very active... Nobody has requested a port for this - are you really personally interested in all these ports?

Reply:
Messengers and i2p? Yes, I am.

(By the way, I don’t think it is a common scenario when users specifically demand a given port. This is not a common scenario in market in general, unless we take some very basic commodity. But anyway, let’s save our time on something better than a trivial argument. I will switch this to tag when I have time to run the build and confirm the app works.)

Reply:
@reneeotten Switched to released version now.